specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/ContainerApps.json

```diff
@@ -551,6 +610,141 @@
           "type": "boolean"
         }
       }
+    },
+    "CustomDomain": {
+      "description": "Custom Domain of a Container App",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Hostname.",
+          "type": "string"
+        },
+        "bindingType": {
+          "description": "Custom Domain binding type.",
+          "enum": [
+            "Disabled",
+            "SniEnabled",
+            "IpBasedEnabled"
```

Comment:
Are we going to support IpBased for containerapps?

Reply:
We will once we support the "reservedIP" feature.

Reply:
If this is not supported for GA then I think it is better not to have it in swagger.

Reply:
removed IPBasedEnabled